COMPATIBILITY.md

```diff
@@ -288,8 +290,10 @@ _When `format=png`_ (default if not specified)
 | summarize(seriesList, intervalString, func='sum', alignToFrom=False)      |
 | threshold(value, label=None, color=None)                                  |
 | timeFunction(name, step=60), Short Alias: time()                          |
+| timeLagSeries(consumeMaxOffsetSeries, produceMaxOffsetSeries)             |
+| timeLagSeriesLists(consumeMaxOffsetSeriesLists, produceMaxOffsetSeriesLists) |
```

Comment:
I don't see the implementation of this function.

Reply:
both functions defined in expr/functions/timeLag. I could've done similar to the way holtwinters logic is shared between two functions, but since they have absolutely the same logic I did it the same way as expr/functions/seriesList